src/lib/Libattr/master_sched_attr_def.xml

```diff
@@ -114,7 +114,7 @@
 	<member_at_comp>comp_str</member_at_comp>
 	<member_at_free>free_str</member_at_free>
 	<member_at_action>NULL_FUNC</member_at_action>
-	<member_at_flags><both>ATR_DFLAG_OPRD | ATR_DFLAG_MGRD | ATR_DFLAG_SvWR</both></member_at_flags>
+	<member_at_flags><both>ATR_DFLAG_OPRD | ATR_DFLAG_MGRD | ATR_DFLAG_MGWR | ATR_DFLAG_SvWR</both></member_at_flags>
```

Comment:
I can address this comment accordingly after getting the reviewers opinion on the following.
https://github.com/PBSPro/pbspro/pull/454#discussion_r151905571

Reply:
Right now the version is communicated from the scheduler to the server on the first scheduling cycle.  You should be able to continue to do this.

Reply:
Have modified the code accordingly.

Reply:
These code changes were required initially when scheduler itself was updating its port back to the server. Since I have modified this now in such a way that admin himself will give sched_port/sched_host through qmgr the above code changes are not required. I just forgot to commit the above file. Please look into my latest commits which I am going to push shortly in a new pull request for the changes.